site/profile/manifests/core/anaconda.pp

```diff
@@ -0,0 +1,78 @@
+# @summary
+#   Installs anaconda
+#
+# @param python_env_name
+#   if exist, sets the name of the virtual environment
+#
+# @param python_env_version
+#   if exist, sets the version of python for the virtual environment
+#
+# @param conda_packages
+#   if contains at least 1 tuple (name and channel), will install the package
+#    
+class profile::core::anaconda (
+  String $python_env_name,
+  String $python_env_version,
+  Tuple   $conda_packages,
+) {
+  case $facts['os']['family'] {
+    'RedHat': {
+      ensure_packages('wget')
```

Comment:
This isn't wrong but as a general practice, it is better to script using `curl` since it is part of the default EL package set.

Reply:
I use the recommendation from the web page, but this can be changed if it is a problem 👍🏻

Reply:
`curl` will always be present so using it instead of wget allows the `ensure_packages()` call to be removed completely.